scheduled-jobs-alert.tf

```diff
@@ -0,0 +1,65 @@
+module "blob-dispatcher-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Reform Blob dispatcher job not running"
```

Comment:
The build has passed with the names with spaces. When the restriction mentioned will take effect? @doncem

Reply:
PR doesn't run terraform code 😉